### ISSUE TITLE
print_tensor only prints three elements

### ISSUE THREAD

**Issue by pablogps** [keras-team/keras#10566]
Using print_tensor prints only the first three elements in a tensor.

Example:
`softmax_layer = keras.print_tensor(softmax_layer, message='My softmax values: ')`        
yields:
`My softmax values:  [[-0.245408952 -0.0407191925 -0.0813238621]...]`

In tf.Print you may control this with the summarize parameter, but print_tensor does not take any such parameters.

Link to my question in StackOverflow: https://goo.gl/GF6TH9

-------------------------------------------------------------------------------------------------------------------------------------------

Please make sure that the boxes below are checked before you submit your issue. If your issue is an implementation question, please ask your question on [StackOverflow](http://stackoverflow.com/questions/tagged/keras) or [join the Keras Slack channel](https://keras-slack-autojoin.herokuapp.com/) and ask there instead of filing a GitHub issue.

Thank you!

- [x] Check that you are up-to-date with the master branch of Keras. You can update with:
pip install git+git://github.com/keras-team/keras.git --upgrade --no-deps

- [x] If running on TensorFlow, check that you are up-to-date with the latest version. The installation instructions can be found [here](https://www.tensorflow.org/get_started/os_setup).

- [ ] If running on Theano, check that you are up-to-date with the master branch of Theano. You can update with:
pip install git+git://github.com/Theano/Theano.git --upgrade --no-deps

- [x] Provide a link to a GitHub Gist of a Python script that can reproduce your issue (or just copy the script here if it is short).

**Comment by HughKu** [comment]
Hi @pablogps, the implementation details of `print_tensor` had been discussed in #3285. It seems like they agreed to keep it as simple as possible with minimal compatibility functionality, so that extra arguments in `tf.Print` like `summarize` in was not incorporated.

**Comment by zwd2016** [comment]
Hi @pablogps, how do you finally change the code to print out all the values of tensor?

**Comment by zwd2016** [comment]
> Using print_tensor prints only the first three elements in a tensor.
> 
> Example:
> `softmax_layer = keras.print_tensor(softmax_layer, message='My softmax values: ')`
> yields:
> `My softmax values: [[-0.245408952 -0.0407191925 -0.0813238621]...]`
> 
> In tf.Print you may control this with the summarize parameter, but print_tensor does not take any such parameters.
> 
> Link to my question in StackOverflow: https://goo.gl/GF6TH9
> 
> Please make sure that the boxes below are checked before you submit your issue. If your issue is an implementation question, please ask your question on [StackOverflow](http://stackoverflow.com/questions/tagged/keras) or [join the Keras Slack channel](https://keras-slack-autojoin.herokuapp.com/) and ask there instead of filing a GitHub issue.
> 
> Thank you!
> 
> * [x]  Check that you are up-to-date with the master branch of Keras. You can update with:
>   pip install git+git://github.com/keras-team/keras.git --upgrade --no-deps
> * [x]  If running on TensorFlow, check that you are up-to-date with the latest version. The installation instructions can be found [here](https://www.tensorflow.org/get_started/os_setup).
> * [ ]  If running on Theano, check that you are up-to-date with the master branch of Theano. You can update with:
>   pip install git+git://github.com/Theano/Theano.git --upgrade --no-deps
> * [x]  Provide a link to a GitHub Gist of a Python script that can reproduce your issue (or just copy the script here if it is short).

Hi @pablogps, how do you finally change the code to print out all the values of tensor?

**Comment by pablogps** [comment]
Hi @zwd2016, apparently it's not really possible. If you really must do this, consider branching the Keras code and editing the print_tensor method to include the extra arguments in Tensorflow. Or perhaps you can call directly the Tensorflow method from Keras, I'm really not sure. Good luck!

**Comment by zwd2016** [comment]
Hi @pablogps, thank you. I am going to try to get this information directly with tf.Print() in TensorFlow.
